[2022-01-30 17:05:38,752] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-30T17:05:35.225150+00:00 [queued]>
[2022-01-30 17:05:38,768] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-30T17:05:35.225150+00:00 [queued]>
[2022-01-30 17:05:38,769] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:05:38,769] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-01-30 17:05:38,769] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-30 17:05:38,791] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-30 17:05:35.225150+00:00
[2022-01-30 17:05:38,797] {standard_task_runner.py:52} INFO - Started process 112 to run task
[2022-01-30 17:05:38,802] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'manual__2022-01-30T17:05:35.225150+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpc2r8exlj', '--error-file', '/tmp/tmpw7zpvmaw']
[2022-01-30 17:05:38,803] {standard_task_runner.py:77} INFO - Job 46: Subtask download_dataset_task
[2022-01-30 17:05:38,892] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task manual__2022-01-30T17:05:35.225150+00:00 [running]> on host 39bc990f24af
[2022-01-30 17:05:39,007] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-30T17:05:35.225150+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-30T17:05:35.225150+00:00
[2022-01-30 17:05:39,009] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-30 17:05:39,009] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv > /opt/***/yellow_tripdata_2021-01.csv']
[2022-01-30 17:05:39,025] {subprocess.py:85} INFO - Output:
[2022-01-30 17:05:50,614] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-30 17:05:50,645] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20220130T170535, start_date=20220130T170538, end_date=20220130T170550
[2022-01-30 17:05:50,705] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-30 17:05:50,749] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
